pkg/sql/logictest/testdata/logic_test/builtin_function

```diff
@@ -102,6 +102,26 @@ SELECT upper('roacH7')
 upper
 ROACH7
 
+query T
+SELECT unaccent('no_special_CHARACTERS1!')
+----
+no_special_CHARACTERS1!
+
+query T
+SELECT unaccent('Żółć')
+----
+Zolc
+
+query T
+SELECT unaccent('⃞h̀ＥＬＬＯ`̀́⃞')
+----
+hELLO`
+
+query T
+SELECT unaccent('some­thing')
```

Comment:
There is actually `U+00AD` in the middle, it's visible in git diff.

Reply:
we can combine these into one test case that's easier to mess with if you do:
```
SELECT unaccent(str) FROM ( VALUES 
   ('no_special_CHARACTERS1!'),
   ('Żółć')
) tbl(str)
```

Reply:
Ok, done